frontend-plugin-core/src/main/java/com/github/eirslett/maven/plugins/frontend/lib/NodeVersionHelper.java

```diff
@@ -223,8 +228,8 @@ public static String getDownloadableVersion(String version) {
 
         version = version.replaceFirst("v", ""); // we're about to add it back
 
-        return findMatchingReleasedVersion(version)
-                .orElse("v" + version);
+//        return findMatchingReleasedVersion(version)
```

Comment:
Leftover?

Reply:
Nah, bug that had to be fixed to ship this. Got a ticket for it and in the commit message